Kentico.Kontent.Management/Models/AssetRenditions/AssetRenditionResponseModel.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Newtonsoft.Json;
+
+namespace Kentico.Kontent.Management.Models.AssetRenditions
+{
+    public class AssetRenditionResponseModel
```

Comment:
Could it be renamed to `AssetRenditionModel` so it follows the naming convention? We don't have ResponseModels suffixes just Model.

Reply:
Sure, renamed.